test_runner/regress/test_storage_controller.py

```diff
@@ -4114,7 +4118,7 @@ def logged_offline():
         ]
     )
 
-    if restart_storcon:
+    if restart_storcon is RestartStorcon.RESTART:
```

Comment:
Why not `==`? `is` returns true if the instances being compared are the same object. It's true in this case, but a bit odd.

Reply:
IDK, probably Rust habit I guess. shrug, will change it to `==`